docs/docs/04-standard-library/01-cloud/endpoint.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Endpoint
+id: endpoint
+description: A resource that represents an endpoint.
+keywords:
+  [
+    Wing reference,
+    Wing language,
+    language,
+    Wing standard library,
+    Wing programming language,
+    Endpoint,
+    URL,
+  ]
+sidebar_position: 1
+---
+
+The `cloud.Endpoint` resource holds a URL and outputs it.
+
+## Usage
+
+```ts playground
+bring cloud;
+
+let endpoint = new cloud.Endpoint("https://example.com");
+```
+
+## Target-specific details
+
+### Simulator (`sim`)
+
+The sim implementation of `cloud.Endpoint` outputs the endpoint URL.
```

Comment:
Outputs where?

I think we also need to add something like `sim.listEndpoints()`